basic-application/basic-application-webapp/src/main/java/fr/openwide/core/basicapp/web/application/common/renderer/ActionRenderers.java

```diff
@@ -0,0 +1,86 @@
+package fr.openwide.core.basicapp.web.application.common.renderer;
+
+import java.util.Locale;
+
+import fr.openwide.core.wicket.more.markup.html.bootstrap.label.model.BootstrapColor;
+import fr.openwide.core.wicket.more.markup.html.bootstrap.label.renderer.BootstrapRenderer;
+import fr.openwide.core.wicket.more.markup.html.bootstrap.label.renderer.BootstrapRendererInformation;
+
+public final class ActionRenderers {
+
+	public static <T> BootstrapRenderer<T> validate() {
+		return new BootstrapRenderer<T>() {
+			private static final long serialVersionUID = 1L;
+			@Override
+			protected BootstrapRendererInformation doRender(T value, Locale locale) {
+				return BootstrapRendererInformation.builder()
+						.label(getString("common.action.validate", locale))
+						.tooltip(getString("common.action.validate", locale))
+						.icon("fa fa-fw fa-check")
+						.color(BootstrapColor.SUCCESS)
+						.build();
+			}
+		};
+	}
+
+	public static <T> BootstrapRenderer<T> add() {
+		return new BootstrapRenderer<T>() {
+			private static final long serialVersionUID = 1L;
+			@Override
+			protected BootstrapRendererInformation doRender(T value, Locale locale) {
+				return BootstrapRendererInformation.builder()
+						.label(getString("common.action.add", locale))
+						.tooltip(getString("common.action.add", locale))
+						.icon("fa fa-fw fa-plus")
+						.color(BootstrapColor.PRIMARY)
+						.build();
+			}
+		};
+	}
+
+	public static <T> BootstrapRenderer<T> remove() {
```

Comment:
I'd be in favor of distinguishing between:
 * `delete`, with an `fa-trash` icon and a "Delete" tooltip ("Supprimer" in French): really erase all information about this object, thus the "trash" icon.
 * `remove`, with an `fa-times` icon and a "Remove" tooltip ("Retirer" in french): put an object out of somewhere, for instance out of a collection, thus the "times" icon, which is, in my opinion, a bit less harsh.

Of course it wouldn't be a rule for every project, but just a good practice for newly started project. Which is the purpose of the basic application, after all.

@florianlacreuse , @gsmet , what do you think?